wallet/src/ui/app/components/receipt-card/index.tsx

```diff
@@ -96,7 +99,8 @@ function ReceiptCard({ tranferType, txDigest }: TxResponseProps) {
                     {AssetCard}
                     {txDigest.amount && (
                         <div className={st.amount}>
-                            {txDigest.amount} <span>{GAS_SYMBOL}</span>
+                            {intl.formatNumber(BigInt(txDigest.amount || 0))}{' '}
```

Comment:
We should add the [balanceFormatingOptions](https://github.com/MystenLabs/sui/blob/39541ca16b83c8aec8d2a2710f5bba79ff178f9d/wallet/src/shared/formatting.ts#L6) as well, like [this](https://github.com/MystenLabs/sui/blob/1f6ce84b165ae3faf8585db7e9373c5e1623da2b/wallet/src/ui/app/components/active-coins-card/index.tsx#L36)  (for all the formats)